vochain/state/state.go

```diff
@@ -73,15 +71,15 @@ type State struct {
 	db db.Database
 	// Store contains the StateDB.  We match every StateDB commit version
 	// with the block height.
-	Store          *statedb.StateDB
+	store          *statedb.StateDB
 	eventListeners []EventListener
 	// Tx must always be accessed via mutex because there will be
 	// concurrent operations on it.  In particular, while the Tx is being
 	// written serially via Tendermint DeliverTx (which updates the StateDB
 	// by processing Vochain transactions), parallel calls of Tendermint
 	// CheckTx happen (which read the temporary state kept in the Tx to
 	// validate Vochain transactions).
-	Tx                treeTxWithMutex
+	tx                treeTxWithMutex
```

Comment:
Why is `tx` now private and how do we get access to the subtrees?

Reply:
so we can be sure no one is using the internal methods of statedb nor tree packages. State should have the API to cover all needs of the vochain application.